pylint/config/config_initialization.py

```diff
@@ -102,6 +103,19 @@ def _config_initialization(
             "unrecognized-option", args=unrecognized_options_message, line=0
         )
 
+    # TODO 3.1: Change this to emit unknown-option-value
```

Comment:
I think this need to be removed from pylint and moved to the config upgraded. Doing those kind of checks at runtime is costly and unnecessary.